integration-tests/manifests/cmd/validate_instrumentation_vars.go

```diff
@@ -17,11 +17,21 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+var appSignalsEnvVarKeys = []string{
+	"OTEL_AWS_APP_SIGNALS_ENABLED",
+	"OTEL_AWS_APPLICATION_SIGNALS_ENABLED",
+	"OTEL_TRACES_SAMPLER_ARG",
+	"OTEL_TRACES_SAMPLER",
+	"OTEL_EXPORTER_OTLP_TRACES_ENDPOINT",
+	"OTEL_AWS_APP_SIGNALS_EXPORTER_ENDPOINT",
+}
+
 func main() {
 
 	args := os.Args
 	namespace := args[1]
 	jsonFilePath := args[2]
+	appSignalsEnabled := args[3] == "app_signals"
```

Comment:
Is args[3] ever not "app_signals"? Might there ever not be a third argument (which would case a panic)?